api/package.json

```diff
@@ -29,9 +29,6 @@
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
     "clean": "tsc --build --clean tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
-    "codecov:browser": "nyc report --reporter=json && codecov -f coverage/*.json -p ../",
-    "codecov:webworker": "nyc report --reporter=json && codecov -f coverage/*.json -p ../",
-    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../",
```

Comment:
Looks like the new codecov is able to search coverage files automatically ([logs](https://github.com/open-telemetry/opentelemetry-js/actions/runs/10995006232/job/30524821179?pr=5011))